### ISSUE TITLE
consider merging mongodb-composable HWRP

### ISSUE THREAD

**Issue by cwjohnston** [edelight/chef-mongodb#166]
http://github.com/cwjohnston/chef-mongodb-composable

**Comment by josephholsten** [comment]
Yeah, I'm going to have to study this a bunch while working on #221 and #194.

I love how `mongodb_instance` just wraps two resources.
